content/tutorials/thematic_maps/thematic_maps.qmd

```diff
@@ -0,0 +1,1435 @@
+---
+title: "Making Thematic Maps"
+author: "Michael Barton"
+date: 2025-06-25
+date-modified: today
+lightbox: true
+engine: knitter
+image: img_thematic/thumbnail.webp
+format: 
+  html:
+    embed-resources: true
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+page-layout: article
+categories: [beginner, intermediate, raster, vector, thematic maps, charts, legend]
+description: This tutorial guides a user through multiple ways of creating thematic maps with raster and vector GIS data.
+execute: 
+  eval: false
+copyright: 
+  holder: Michael Barton
+  year: 2025
+funding: "Creation of this tutorial was supported in part by US National Science Foundation grant FAIN 2303651."
```

Comment:
This one doesn't show up in the text, so we need to put it there explicitely, I will make a suggestion at the bottom of the page.

Reply:
OK. Done